### ISSUE TITLE
Toggle CLI turns brightness up to 100%

### ISSUE THREAD

**Issue by filmgarage** [linusg/rpi-backlight#44]
If the screen is on and the command power off is sent, the screen is back to the former brightness when turned on again.

For example:
```
// Brightness 20%
rpi-backlight --p off //screen blank
rpi-backlight --p on // screen 20%
```

Toggle does not remember the former brightness:

```
// Brightness 20%
rpi-backlight --p toggle //screen blank
rpi-backlight --p toggle // screen 100%
```

Is this intentional? I would prefer the previous setting to be remembered.

**Comment by linusg** [comment]
Yep, that was recently discussed here: https://github.com/linusg/rpi-backlight/pull/40

I consider it a mistake and agree with

> I would prefer the previous setting to be remembered.

But apparently it slipped through in the past. A fix would be very welcome!

**Comment by p1r473** [comment]
Hi Linus,
My pull request included a fix for this issue

**Comment by linusg** [comment]
@p1r473 feel free to reopen it then! I generally don't do that myself.

**Comment by p1r473** [comment]
https://github.com/linusg/rpi-backlight/pull/68 should fix this
